e2e/react-router/basic-virtual-named-export-config-file-based/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
do we really need a  separate e2e test for this?  isn't the generator test sufficient?

Reply:
Two reasons on my part:
1. There already was a virtual routes example using the config directly supplied by the user, so this would also validate that the file reading was working.
2. I wanted a test, without the `routeTree.gen.ts` committed into git, so that on every CI run, it generated the `routeTree.gen.ts` on it's and ensured the navigations were successful.